Files/Filesystem/FilesystemOperations/IFilesystemOperations.cs

```diff
@@ -46,7 +46,7 @@ public interface IFilesystemOperations : IDisposable
         /// </returns>
         Task<IStorageHistory> CopyAsync(IStorageItem source,
                                         string destination,
-                                        IProgress<float> progress,
+                                        IProgress<double> progress,
```

Comment:
Any reason to switch from float to double? It's not like we need that precision :)